assets/javascripts/discourse/components/admin/filter-children-form.hbs

```diff
@@ -0,0 +1,52 @@
+{{#if (eq @toggleChildMenuFor @filterTag.name)}}
```

Comment:
We check which global filter the menu was toggled for so that we only open one at a time.